ui-v2/app/templates/components/healthchecked-resource.hbs

```diff
@@ -25,7 +25,7 @@
         <li class="passing">
             <a href={{href}}>
                 <strong>{{healthy.length}}</strong>
-                <span>other passing {{pluralize-word healthy.length 'check' omitCount=true}}</span>
+                <span>other passing {{pluralize healthy.length 'check' without-count=true}}</span>
```

Comment:
Kebab casing, huh? Are you trying to go for that native feel?

Reply:
This was just a consequence of switching from `ember-pluralize` to `ember-inflector`

`{{pluralize-word}}` https://github.com/rsocci/ember-pluralize#usage
`{{pluralize}}` https://github.com/emberjs/ember-inflector#template-helpers